Makefile

```diff
@@ -20,8 +20,8 @@ SETTINGS_TIMESTAMP = $(TIMESTAMPS)/.settins.timestamp
 DOCKER_BUILD_TIMESTAMP = $(TIMESTAMPS)/.docker-test.timestamp
 
 # Docker variables
-DOCKER_IMG_LOCAL_TAG = swisstopo/$(SERVICE_NAME):latest
-DOCKER_IMG_LOCAL_TAG_TEST = swisstopo/$(SERVICE_NAME):latest-dev
+DOCKER_IMG_LOCAL_TAG = swisstopo/$(SERVICE_NAME):$(USER).latest
+DOCKER_IMG_LOCAL_TAG_TEST = swisstopo/$(SERVICE_NAME):$(USER).latest-dev
```

Comment:
Hmm, this doesn't play well with `docker-compose.yml`, it would be nice to be able to test the `docker-compose.yml` locally

Reply:
Well this tag is only used when building locally and doesn't have anything to do with docker-compose.yml. This latter uses its own build system and tag, therefore it shouldn't be an issue. But I'll test it to be sure.

Reply:
@boecklic I tried the ``docker-compose up` locally and it worked, so I don't really understand the issue here with using the `$(USER)` in makefile for the `dockerbuild` targets ? The advantage of using the `$(USER)` in local tag is that your can still then manually push it to dockerhub in order to test the image on k8s dev and it gives a hint to people from where come this local image.

Reply:
It's just that docker-compose will download the `:latest-dev` and not the locally built image. If you need to adapt docker-compose (cause e.g. you change sth. in the Dockerfile) you'll want that it uses the locally built image. But we can leave it as you have it now, we just need to remember that when adapting Dockerfile and testing if the CI will work with docker-compose will require temporary manual changes